lib/matplotlib/patches.py

```diff
@@ -127,6 +127,29 @@ def contains(self, mouseevent, radius=None):
         """
         Test whether the mouse event occurred in the patch.
 
+        Parameters
+        ----------
+        mouseevent : .MouseEvent
```

Comment:
```suggestion
        mouseevent : `~matplotlib.backend_bases.MouseEvent`
```
(But at least single backticks.)